src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -353,6 +355,21 @@ static void window_editor_object_selection_select_default_objects()
     }
 }
 
+static void select_designer_objects()
```

Comment:
All new functions are to be UpperCamelCase

Reply:
It's done as of now. The new function is SelectDesignerObjects().